utils.py

```diff
@@ -55,14 +82,31 @@ def summarize_report(df_shares, df_forex, df_dividends, df_fees, df_taxes):
     # for tax reasons, we usually also want the sum of gains
     # and the sum of losses
     share_gain_series = df_shares["Gain [EUR]"]
-    share_losses = share_gain_series[share_gain_series < 0].sum()
-    share_gains = share_gain_series[share_gain_series > 0].sum()
+    if share_gain_series[share_gain_series < 0].shape[0] > 0:
+        share_losses = share_gain_series[share_gain_series < 0].sum()
```

Comment:
Are these `.sum()` outputs already Decimals? If so, I may not understand why the distinction between the `< 0` and `> 0` branches is necessary.

Reply:
`.sum()` is a Decimal, unless the DataFrame is empty because the sum of an empty DataFrame is float(0) unfortunately.